submissions/views.py

```diff
@@ -136,31 +147,36 @@ def form_valid(self, form):
 
 @xframe_options_exempt
 @csrf_exempt
-def panel(request):
+def panel(request, conslug="ConFusion2020"):
```

Comment:
Is `"Confusion2020"` supposed to be hard-coded here or should it just be `conslug`?

Reply:
yup, this is the default value, which will get overridden by the value passed in from the url. Anytime a python function definition has a variable defined inside it, that's an optional default value. Function calls can pass in something to override it, or they can pass in nothing and get the default.

For the last ConFusion we didn't have the con slug in the URL, so those pages (which just say "the form is closed") would break if I didn't set a default to say "no slug means it was ConFusion 2020." At some point when we're less busy, I will ask the web team to update the 2020 forms to include the slug, and then we can change or remove the default.

Another way to handle this would be to have slugless urls route to a page that is a generic "this form isn't available" page. That way, I wouldn't need to make another code change once the 2020 forms are fixed. But that would have involved writing more code to set up a new page and work out what the default page would look like--it's probably worth doing in the future but for now, setting the default is the quickest way to solve the problem. And since the pages are set to 404 if the slug in the url can't be found, this hardcoding shouldn't cause any problems for any instances of the app that don't have ConFusion2020 in their DB.